scripts/ci/images/ci_prepare_ci_image_on_ci.sh

```diff
@@ -37,7 +37,7 @@ function build_ci_image_on_ci() {
         if [[ ${GITHUB_REGISTRY_PUSH_IMAGE_TAG} != "latest" ]]; then
             python_tag_suffix="-${GITHUB_REGISTRY_PULL_IMAGE_TAG}"
         fi
-        # first we pull base python image. We will need it to re-push it after master build
+        # first we pull base Python image. We will need it to re-push it after master build
```

Comment:
Ones like this are referring to `docker pull python` - so not sure it makes sense here.